styles/modules/_userPage.scss

```diff
@@ -20,11 +20,16 @@
       padding: 268px 0 40px 0;
     }
 
+    .listBox {
+      .listItem {
+        padding: $pad 0;
+      }
+    }
+
```

Comment:
This is needed because of other changes made to the listbox, rather than layering on more changes, it would be better to fix the original problem caused by adding too much padding to the container.

I'll put in a patch.